BuildTools/Package.swift

```diff
@@ -0,0 +1,37 @@
+// swift-tools-version:6.0
+import Foundation
+import PackageDescription
+
+let package = Package(
+    name: "BuildTools",
+    platforms: [.macOS(.v10_13)],
+    dependencies: [
+        .package(url: "https://github.com/SimplyDanny/SwiftLintPlugins", exact: loadSwiftLintVersion()),
```

Comment:
Does the version numbers have to be in `.swiftlint.yml`? Are there any other roadblocks for simply defining the version here in the package:

```
.package(url: "https://github.com/SimplyDanny/SwiftLintPlugins", exact: "0.58.2"),
```

Reply:
`.swiftlint.yml` needs to specify the SwiftLint version the configuration expects

https://github.com/Automattic/a8c-ci-toolkit-buildkite-plugin/blob/29064ab1e6e49135f9700d86b7b3279081b9ebc9/bin/run_swiftlint#L23-L27

We don't _have to_ read the version from there here, but I find it nice to have a single source of truth.

We could duplicate the value here, but we _might_ run into some annoying issues if the version defined here diverges from the one in the config so that we end up with CI running a different version than local and giving different feedback.